chain/actors/builtin/market/v0.go

```diff
@@ -0,0 +1,25 @@
+package market
+
+import (
+	"github.com/filecoin-project/specs-actors/actors/builtin/market"
+	"github.com/filecoin-project/specs-actors/actors/util/adt"
+)
+
+type v0State struct {
+	market.State
+	store adt.Store
+}
```

Comment:
Consider an interface assertion here?

Reply:
That _shouldn't_ really be necessary as `Load` will assert the interface.